grails-app/assets/javascripts/streama/services/api-service.js

```diff
@@ -329,7 +335,11 @@ angular.module('streama').factory('apiService', function ($http, $rootScope, con
 
 			listRecommendations: function () {
 				return $http.get('dash/listRecommendations.json');
-			}
+			},
+
+      listWatchlistEntries: function (params) {
+        return $http.get('dash/listWatchlistEntries.json', {params: params});
```

Comment:
Should this endpoint maybe just be under watchlistEntry/list instead of dash/listWatchlistEntries?